src/datastore/postgres/schema/v7.ts

```diff
@@ -0,0 +1,10 @@
+import { IDatabase } from "pg-promise";
+import { MatrixUser } from "matrix-appservice-bridge";
+
+// tslint:disable-next-line: no-any
+export async function runSchema(db: IDatabase<any>) {
+    // Drop constraints
+    await db.none(`
+        ALTER TABLE puppets DROP CONSTRAINT puppets_slackteam_key;
+        ALTER TABLE puppets DROP CONSTRAINT puppets_matrixuser_key;`);
```

Comment:
I can't find the code that created these constraints.

`cons_puppets_uniq` is the only constraint on the table `puppets`.
https://github.com/matrix-org/matrix-appservice-slack/blob/bf18f238a925e85d39c6b3e68d0ee154d978d3fd/src/datastore/postgres/schema/v2.ts#L10

Reply:
https://github.com/matrix-org/matrix-appservice-slack/blob/hs/fix-puppet-constraint/src/datastore/postgres/schema/v2.ts#L6-L8

`UNIQUE` is essentially a shorthand form for writing constraints. To remove `UNIQUE`, you need to drop it.